liferay-lima/liferay-portal
# LPS-166653 Allow same url title for different translations of friendly url

# What is this about?

Allow to use the same url title for different translations for the same friendly url.

# How do I test this?

You can test it in 2 different places: journal and pages. For both, the steps a the following:
1. Create an entry with url title "abc" for the default language.
2. Edit the entry, add a url title translation for another language and put the same value ("abc").

This should *not* fail; both languages should have the same url title.

This should only work for the same entry; trying to reuse the url title for other entry should fail.
